qemu/qmp.go

```diff
@@ -727,6 +727,51 @@ func (q *QMP) ExecuteBlockdevDel(ctx context.Context, blockdevID string) error {
 	return q.executeCommand(ctx, "x-blockdev-del", args, nil)
 }
 
+// ExecuteNetdevAdd adds a Net device to a QEMU instance
+// using the netdev_add command. netdevID is the id of the device to add.
+// Must be valid QMP identifier.
+func (q *QMP) ExecuteNetdevAdd(ctx context.Context, netdevType, netdevID, ifname, downscript, script string, queues int) error {
+	args := map[string]interface{}{
+		"type":       netdevType,
+		"id":         netdevID,
+		"ifname":     ifname,
+		"downscript": downscript,
```

Comment:
Any reason `script` is not exposed but `downscript` is? In kata containers, I think we will always set them to "no" but if we expose one of them, we'd better expose the other too.

Reply:
"script" is in the next line 739

Reply:
sorry I thought I deleted that line of comment but somehow it still appeared there...